### PR TITLE
[GraphBolt] update URL for ogb-lsc-mag240m

### PR DESCRIPTION
**Pull request by Rhett-Ying**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Review by frozenbugs — APPROVED** [review]

File: python/dgl/graphbolt/impl/ondisk_dataset.py [hunk 1]
```diff
@@ -528,23 +528,34 @@ class BuiltinDataset(OnDiskDataset):
         The root directory of the dataset. Default ot ``datasets``.
     """
 
+    # For dataset that is smaller than 30GB, we use the base url.
+    # Otherwise, we use the accelerated url.
     _base_url = "https://data.dgl.ai/dataset/graphbolt/"
+    _accelerated_url = (
+        "https://dgl-data.s3-accelerate.amazonaws.com/dataset/graphbolt/"
+    )
     _datasets = [
```
Comment (frozenbugs) [inline python/dgl/graphbolt/impl/ondisk_dataset.py:537]: I would rename this to _small_datasets

Comment (frozenbugs) [reply python/dgl/graphbolt/impl/ondisk_dataset.py:537]: and `_datasets = _small_datasets + _large_datasets`

Comment (Rhett-Ying) [reply python/dgl/graphbolt/impl/ondisk_dataset.py:537]: actually ,it's small+medium vs large

Comment (frozenbugs) [reply python/dgl/graphbolt/impl/ondisk_dataset.py:537]: either way works, just a nit :)

**Pull request merged**